pkg/idaes_examples/common/hda/ideal_VLE.py

```diff
@@ -636,6 +713,7 @@ def _energy_internal_mol_phase_comp(self):
         self.energy_internal_mol_phase_comp = Var(
                 self._params.phase_list,
                 self._params.component_list,
+                units=pyunits.J/pyunits.mol,
                 doc="Phase-component molar specific internal energies [J/mol]")
```

Comment:
Minor comment, but I would suggest we remove the units from the doc strings now that we have explicitly defined units.